ci/docker/runtime_functions.sh

```diff
@@ -1219,7 +1219,7 @@ build_docs() {
 
     # copy the full site for this version to versions folder
     mkdir -p html/versions/master
-    for f in 404.html api assets blog community ecosystem features feed.xml get_started index.html; do
+    for f in 404.html api assets blog community ecosystem features feed.xml get_started index.html .asf.yaml; do
```

Comment:
The `.asf.yaml` would already be in `html/`. Do we need to copy it to `html/versions/master`?

Reply:
Thanks for pointing it out. If .asf.yaml is already in html/ then we do not need to pack it into versions/master again.